templates/deno/src/permissions.ts.twig

```diff
@@ -0,0 +1,22 @@
+export class Permissions {
+
+    static read = (role: string[]): string => {
+        return `read("${role}")`
+    }
+
+    static write = (role: string[]): string => {
```

Comment:
Is role a string array ? 
Plus - refer to my comment on Kotlin. We should pass `Role` as the parameter type instead of the string

Reply:
done